models/Tweet.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const {ObjectId} = mongoose.Schema.Types
+const TweetSchema = new mongoose.Schema({
+    
+    content:{
+        type: String,
+        required: true,
+    },
+    likes:[
```

Comment:
Have you thought about what happens when the likes grow uncontrollably? Like when they are 4000 or so?

What happens if you need to retrieve all the information of the users that liked a tweet? Do you not think having to `populate` 4000 records is too expensive?

Reply:
I will make some research and make the corrections

Reply:
Your research could be focused on many-to-many relationship in Mongo